app/build.gradle

```
@@ -77,6 +77,10 @@ dependencies {
     runtimeOnly 'io.jsonwebtoken:jjwt-impl:0.11.5'
     runtimeOnly 'io.jsonwebtoken:jjwt-jackson:0.11.5'
 
+    // azure sdk
+    implementation 'com.azure:azure-security-keyvault-secrets:4.5.4'
```

See if this fixes the vulnerability.

```suggestion
    implementation 'com.azure:azure-security-keyvault-secrets:4.6.0'
```

---

Good idea, trying the update.